tests/cpp/test_host_irs.cpp

```diff
@@ -708,6 +708,43 @@ INSTANTIATE_TEST_SUITE_P(
       return ss.str();
     });
 
+using MatmulHostIrTest = NVFuserTest;
+
+TEST_F(MatmulHostIrTest, HostIr) {
```

Comment:
```suggestion
TEST_F(HostIrTest, Matmul) {
```
I understand HostIrTest is a parameterized test, but it's fine to use it as a TEST_F as long as we don't call GetParam.

Reply:
I tried that previously but had an error: https://github.com/NVIDIA/Fuser/pull/2491#discussion_r1660893569

Reply:
let me know if I'm missing something